src/content/core/okteto-variables.mdx

```diff
@@ -197,13 +197,34 @@ USERNAME has a static value, and PASSWORD is dynamically generated.
 Okteto provides a list of variables that serve as feature flags for some of the features we release in our product. Below is a list of those variables that you can use to leverage these newer and experimental features:
 
 - `OKTETO_SMART_BUILDS_ENABLED`: enables the "smart builds" feature that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times.
-- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`:
-- `OKTETO_KUBERNETES_TIMEOUT`:
-- `OKTETO_HOME`:
-- `OKTETO_FORCE_REMOTE`:
-- `OKTETO_FOLDER`:
-- `OKTETO_REMOTE_CLI_IMAGE`:
+- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`: Compose services mounting the same volume will be placed on the same node using Kubernetes's pod affinity. (Defaults to `true`)
+- `OKTETO_KUBERNETES_TIMEOUT`: specifies the timeout while deploying. (defaults to `0`)
+- `OKTETO_FORCE_REMOTE`: if true deploy the repository on the cluster vs locally.
+- `OKTETO_FOLDER`: defines the path where the okteto folder is located (defaults to `$OKTETO_HOME/.okteto`).
+- `OKTETO_REMOTE_CLI_IMAGE`: specifies the CLI image that Okteto is going to use while deploying/destroying on remote.
 - `OKTETO_LOCAL_REGISTRY_STORE_ENABLED`: enables the local registry credentials over the credentials stored in the cluster. (Defaults to `true`) 
+- `OKTETO_BIN`: version of the CLI that the init containers will use.
+- `OKTETO_SKIP_CLEANUP`: if set skips the deletion of the syncthing folder when running `okteto down`
+- `OKTETO_USE_STATIC_KUBETOKEN`: use a fixed token rather than a dynamic token for interactions with the Kubernetes API. (Defaults to `false`)
+- `OKTETO_HOME`: defines the path of okteto folder (Defaults to `$HOME`)
+- `OKTETO_URL`:  specifies the Okteto cluster URL to connect to.
+- `OKTETO_TOKEN`: specifies the Okteto token that should be used to connect to the Okteto cluster.
+- `OKTETO_CONTEXT`: specifies the Kubernetes cluster name to connect to.
+- `OKTETO_NAMESPACE`: specifies the Kubernetes namespace that the CLI will operate.
+- `COMPOSE_FILE`: specifies the compose file that the user has to use.
```

Comment:
What's the use case for this instead of using `-f`? I don't think we should document this as a feature flag